raiden/ui/cli.py

```diff
@@ -745,6 +754,20 @@ def run(ctx, **kwargs):
     print('Welcome to Raiden, version {}!'.format(get_system_spec()['raiden']))
     from raiden.ui.console import Console
     from raiden.api.python import RaidenAPI
+    from raiden.utils import address_checksum_and_decode
+    import toml
+
+    if kwargs['config_file']:
+        d = toml.load(kwargs['config_file'])
```

Comment:
This can go wrong and raise an exception. Please handle these. Ref: https://pypi.org/project/toml/

Reply:
i handled the exception in the new commit, but i till think let it crash in this case is better since user may not notice the warning log which leading to unexpected params